service-rpc/src/main/thrift/rpc.thrift

```diff
@@ -304,5 +309,7 @@ service TSIService {
 
 	TSRPCResp insertRow(1:TSInsertReq req);
 
+	TSRPCResp deleteTimeseries(1:TSDeleteReq req);
```

Comment:
deleteTimeseries means deleting all data and schema of a timeseries, this should be called delete

Reply:
It would cause error message in thrift because 'delete' is the reserved word, which would show the message below:
```
[ERROR] thrift failed error: [ERROR:C:/Users/asus/Documents/Java/incubator-iotdb/service-rpc/src/main/thrift/rpc.thrift:312] (last token was 'delete')
Cannot use reserved language keyword: "delete"
```

Reply:
I have changed 'deleteTimeseries' into 'deleteRow'. What do you think?

Reply:
deleteData is better